buildSrc/src/main/java/Dependencies.kt

```diff
@@ -3,13 +3,13 @@ import org.gradle.api.artifacts.dsl.RepositoryHandler
 @Suppress("unused")
 object Versions {
     const val compileSdkVersion = 31
-    const val minSdkVersion = 23
+    const val minSdkVersion = 25
```

Comment:
Wondering if this min sdk version is too high. Maybe we can look at what percentage of devices we would lose with this bump

Reply:
It's probably fine, marshmallow is 3.5%, Lollipop 2.9% and Kitkat 1%.  By the time we actually release this those will be even lower and/or we can revisit later

Reply:
I thought about it but don't think it will make a big difference by the time it actually gets release on the play store